### PR TITLE
Added Insert-IT as a service provider

### PR DESCRIPTION
**Pull request by derDeno**
## Added Insert-IT as a new service provider. 
This provides a streamlined way for all Insert-IT based apps to be integrated.

Note: The benefit of this integration is the easy configuration, as it could be used with a location ID, which needs to be extracted from an url, or by simply providing street name and house number. Therefore it is easier to integrate than using static ics files.

### Currently these municipalities are supported:
- Hattingen, DE
- Herne, DE
- Kassel, DE
- Krefeld, DE
- Mannheim, DE
- Offenbach, DE

### Breaking Change
This replaces the offenbach_de source and fixes the broken mannheim source.

### Testing
Test where done successfully:
```
Testing source insert_it_de ...
  found 136 entries for Offenbach Address
  found 136 entries for Offenbach Location ID
  found 136 entries for Mannheim Address
  found 136 entries for Mannheim Location ID
```

### Misc
Two broken docs where also fixed (missing .md):
- lund_se.md
- stoke_gov_uk.md

### CODE REVIEW

File: doc/source/insert_it_de.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        municipality: Offenbach
+        street: Kaiserstraße
+        hnr: 1
+```
+
+
+## How to get the source arguments from website
+
+Another way get the source arguments is to us a (desktop) browser, e.g. Google Chrome:
+Easy way to get the source arguments is to use a (desktop) browser, eg.g Google Chrome:
```
Comment (5ila5) [inline doc/source/insert_it_de.md:62]: duplicate message

File: doc/source/insert_it_de.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+**hnr**  
+*(integer) (optional)*
+
+**location_id**  
+*(integer) (optional)*
+
+
+Either `street` and `hnr` or `location_id` is required. Both could also be set, but `location_id` is prioritized.
+If none is set, it will fail.
+
```
Comment (5ila5) [inline doc/source/insert_it_de.md:33]: maybe add a list of supported municipality arguments

**Review by 5ila5 — CHANGES_REQUESTED** [review]

**Comment by derDeno** [comment]
requested changes were done

**Review by 5ila5 — APPROVED** [review]

**Comment by 5ila5** [comment]
Thanks for your contribtuion

**Pull request merged**